omniscidb/Tests/IntelGPUEnablingTest.cpp

```diff
@@ -1001,6 +1001,13 @@ int main(int argc, char* argv[]) {
                      "Dump IR and PTX for all executed queries to file."
                      " Currently only supports single node tests.");
 
+  desc.add_options()("dump-after-all",
```

Comment:
Can we have it in `ConfigBuilder` so that it would be more widely available? It was supposed to replace all these options in each separate test binary.